std/python/Lib.hx

```diff
@@ -36,7 +36,7 @@ class Lib {
 	/**
 		Print the specified value on the default output.
 	**/
-	public static function print(v:Dynamic):Void {
+	public static function print_(v:Dynamic):Void {
```

Comment:
I don't think we want to rename the `print` function as discussed in https://github.com/HaxeFoundation/haxe/issues/6183.